### PR TITLE
Add pgmajfault to data returned by vmstat on Linux

### PR DESCRIPTION
**Pull request by james-bebbington**
Minor change to add `pgmajfault` to swap data returned by `vmstat` on Linux

### CODE REVIEW

**Comment by shirou** [comment]
sorry for late response. Maybe I'm wrong, but I think `pgmajfault` is only in a Linux. if so, please add a comment ` // Linux specific numbers` like `VirtualMemoryStat`.

File: mem/mem.go [hunk 1]
```diff
@@ -88,6 +88,7 @@ type SwapMemoryStat struct {
 	PgIn        uint64  `json:"pgin"`
 	PgOut       uint64  `json:"pgout"`
 	PgFault     uint64  `json:"pgfault"`
+	PgMajFault  uint64  `json:"pgmajfault"`
```
Comment (james-bebbington) [inline mem/mem.go:91]:
```suggestion
	// Linux specific numbers
	// https://www.kernel.org/doc/Documentation/cgroup-v2.txt
	PgMajFault  uint64  `json:"pgmajfault"`
```

**Comment by james-bebbington** [comment]
> sorry for late response. Maybe I'm wrong, but I think `pgmajfault` is only in a Linux. if so, please add a comment ` // Linux specific numbers` like `VirtualMemoryStat`.

No worries - done

**Comment by shirou** [comment]
Thank you so much!

**Pull request merged**